Lib/test/test_posix.py

```diff
@@ -283,6 +307,34 @@ def test_pwrite(self):
         finally:
             os.close(fd)
 
+    @unittest.skipUnless(hasattr(posix, 'pwritev'), "test needs posix.pwritev()")
+    def test_pwritev(self):
+        fd = os.open(support.TESTFN, os.O_RDWR | os.O_CREAT)
+        try:
+            n = os.pwritev(fd, [b'test1tt2t3'],2)
```

Comment:
I would prefer to test writing multiple buffers than a single buffer. Maybe write [b'test1', b'tt2', b't3']?

I would prefer to write explicitly the first 2 bytes, but then seek back at offset 0 before calling os.pwritev(). I don't want to test in Python the behaviour of pwritev() for non-offset offset with an empty file. Python should test its binding, not the exact behaviour of the OS.

For example:
write(b'XX')
seek(0)
pwrite(..., offset=2)
assert read() == b'XX...'